packages/manager/apps/web/client/app/private-database/whitelist/private-database-whitelist.html

```diff
@@ -10,82 +10,98 @@
+                        data-title="'privateDatabase_tabs_whitelist_list_ip_network' | translate"
+                        data-property="ip"
+                        data-sortable
+                    ></oui-datagrid-column>
+                    <oui-datagrid-column
+                        data-title="'privateDatabase_tabs_whitelist_list_service' | translate"
+                        data-property="service"
+                    >
+                        <i
+                            class="oui-icon oui-icon-success"
+                            data-ng-show="$row.service"
+                            aria-hidden="true"
+                        ></i>
+                    </oui-datagrid-column>
+                    <oui-datagrid-column
+                        data-title="'privateDatabase_tabs_whitelist_list_sftp' | translate"
+                        data-property="sftp"
+                    >
+                        <i
+                            class="oui-icon oui-icon-success"
+                            data-ng-show="$row.sftp"
+                            aria-hidden="true"
+                        ></i>
+                    </oui-datagrid-column>
+                    <oui-datagrid-column
+                        data-title="'privateDatabase_tabs_whitelist_list_status' | translate"
+                        data-property="name"
+                    >
+                        <span
+                            class="label label-info"
+                            data-ng-if="$row.creating"
+                            data-translate="privateDatabase_tabs_whitelist_list_status_creating"
                         ></span>
-                    </oui-action-menu-item>
-                    <oui-action-menu-item
-                        data-on-click="listCtrl.deleteWhitelist($row)"
-                        ><span
-                            data-translate="privateDatabase_tabs_whitelist_list_popover_delete"
+                        <span
+                            class="label label-info"
+                            data-ng-if="$row.updating"
+                            data-translate="privateDatabase_tabs_whitelist_list_status_updating"
                         ></span>
-                    </oui-action-menu-item>
-                </oui-action-menu>
-            </oui-datagrid>
-        </div>
-
-        <div class="col-md-3 mt-5 mt-lg-0">
-            <button
-                class="btn btn-block btn-default"
-                type="button"
-                data-translate="privateDatabase_tabs_whitelist_list_add_whitelist"
-                data-ng-click="listCtrl.createWhitelist()"
-            ></button>
+                        <span
+                            class="label label-info"
+                            data-ng-if="$row.deleting"
+                            data-translate="privateDatabase_tabs_whitelist_list_status_deleting"
+                        ></span>
+                        {{ $row.name | wucSliceContent: 50 }}
```

Comment:
```suggestion
                       <span data-ng-bind=$row.name | wucSliceContent: 50"></span>
```

Reply:
Seen together, code was only moved. We will leave this for now.